catroid/src/main/java/org/catrobat/catroid/content/bricks/BrickBaseType.java

```diff
@@ -74,41 +98,26 @@ public void setCheckboxView() {
 			enabled = checkbox.isEnabled();
 			isChecked = checkbox.isChecked();
 		}
+
 		checkbox = view.findViewById(R.id.brick_checkbox);
 		checkbox.setChecked(isChecked);
 		checkbox.setVisibility(checkboxVisibility);
 		checkbox.setEnabled(enabled);
-
-		final Brick instance = this;
 		checkbox.setOnClickListener(new View.OnClickListener() {
 			@Override
 			public void onClick(View view) {
 				adapter.handleCheck(instance, ((CheckBox) view).isChecked());
```

Comment:
You can use `BrickBaseType.this` instead of creating a local variable.

Reply:
thanks for the hint -> I just copied it from the old version. For now I'm not really changing too much, so there could definitely be more stuff like this :wink: .